README.md

```diff
@@ -120,6 +120,20 @@ mod ffi {
 }
 ```
 
+## Comparison to other Rust Qt bindings
+
+| project |  integrate into C++ codebase | safe Rust | maintained* | binding mechanism |
+|-------- | ---------------------------- | --------- | ----------- | ----------------- |
+| cxx-qt  |  ✔                           | ✔        |  ✔          | [cxx](https://cxx.rs) plus additional code generation to implement QObject subclasses in Rust and bind them to C++ |
```

Comment:
We don't state where CXX-Qt is *not* suited ?

Reply:
QWidgets, but kinda... it should still be feasible to use cxx-qt to implement Qt models in Rust within a QWidgets application even if we don't have bindings for QWidgets in cxx-qt-lib

Reply:
Qt Widgets should be fine, as CXX-Qt is just trying to add enough syntax to CXX so that you can describe a QObject from Rust. Eg we allow you to define properties, signals, invokables (for QML), slots, base classes etc. And by using the new syntax you should be able to describe existing widgets API and objects that you need to expose across. A few more common types could be added to cxx-qt-lib (eg QByteArray) but I don't think we would need to add eg QPushButton.

But this comes to the area where CXX-Qt is weaker, in CXX-Qt you mainly create a split between your UI layer and your business logic layer. Then you should use models or QObject's from CXX-Qt to describe state. And finally then use the existing QWidgets/C++ or QML/JS API for describing the UI. Some of the other bindings however allow you to write the UI layer in "Rust" as well (although they tend to just be a wrapped C++ API).